src/System.Net.Requests/tests/HttpWebRequestTest.cs

```diff
@@ -301,7 +317,8 @@ public void TimeOut_SetThenGet_ValuesMatch(Uri remoteServer)
                 var sw = Stopwatch.StartNew();
                 WebException exception = Assert.Throws<WebException>(() =>
                 {
-                    var response = (HttpWebResponse)request.GetResponse();
+                    var response = request.GetResponse();
+                    response.Dispose();
```

Comment:
True as long as GetResponse throws WebException which is the intent of this test. If not, it should still do cleanup